nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/NonDexSurefireExecution.java

```diff
@@ -102,6 +118,10 @@ private Xpp3Dom addExcludedGroups(Xpp3Dom configNode) {
             if ("excludedGroups".equals(config.getName())) {
                 Logger.getGlobal().log(Level.INFO, "Adding excluded groups to existing ones");
                 String current = config.getValue();
+                // Should not add duplicate entry for NonDexIgnore
+                if (current.contains("edu.illinois.NonDexIgnore")) {
```

Comment:
Because at some point it was added to those pom files. The correct way would have been to add brand new tests. Otherwise, our ITs would not catch bugs that only appear when the *Ignore is not used. There are tests that have *Ignore in them.